src/features/common/Layout/ProjectPropsContext.tsx

```diff
@@ -1,127 +1,167 @@
+    React.useState<boolean>(false);
+  const [searchedProject, setsearchedProjects] = React.useState<
+    SearchProject[]
+  >([]);
+  const [geoJson, setGeoJson] = React.useState<FeatureCollection<
+    Geometry,
+    GeoJsonProperties
+  > | null>(null);
+  const [siteExists, setsiteExists] = React.useState<boolean>(false);
+  const [selectedSite, setSelectedSite] = React.useState<number>(0);
   const infoRef = React.useRef(null);
   const exploreContainerRef = React.useRef(null);
-  const [exploreExpanded, setExploreExpanded] = React.useState(false);
-  const [exploreForests, setExploreForests] = React.useState(false);
-  const [explorePotential, setExplorePotential] = React.useState(false);
-  const [exploreDeforestation, setExploreDeforestation] = React.useState(false);
-  const [explorePlanted, setExplorePlanted] = React.useState(false);
-  const [infoExpanded, setInfoExpanded] = React.useState(null);
-  const [openModal, setModalOpen] = React.useState(false);
-  const [isMobile, setIsMobile] = React.useState(false);
-  const [satellite, setSatellite] = React.useState(false);
-  const [filteredProjects, setFilteredProjects] = React.useState(null);
-  const [filtersOpen, setFilterOpen] = React.useState(false);
-  const [purpose, setPurpose] = React.useState('trees');
-  const [plantLocationsLoaded, setPlantLocationsLoaded] = React.useState(false);
+  const [exploreExpanded, setExploreExpanded] = React.useState<boolean>(false);
+  const [exploreForests, setExploreForests] = React.useState<boolean>(false);
+  const [explorePotential, setExplorePotential] =
+    React.useState<boolean>(false);
+  const [exploreDeforestation, setExploreDeforestation] =
+    React.useState<boolean>(false);
+  const [explorePlanted, setExplorePlanted] = React.useState<boolean>(false);
+  const [infoExpanded, setInfoExpanded] = React.useState<string | null>(null);
+  const [openModal, setModalOpen] = React.useState<boolean>(false);
+  const [isMobile, setIsMobile] = React.useState<boolean>(false);
+  const [satellite, setSatellite] = React.useState<boolean>(false);
+  const [filteredProjects, setFilteredProjects] = React.useState<
+    SearchProject[] | null
+  >(null);
+  const [filtersOpen, setFilterOpen] = React.useState<boolean>(false);
+  const [purpose, setPurpose] = React.useState<string>('trees');
+  const [plantLocationsLoaded, setPlantLocationsLoaded] =
+    React.useState<boolean>(false);
   const { embed, showProjectList } = React.useContext(ParamsContext);
 
   const mapRef = React.useRef(null);
+
   const EMPTY_STYLE = {
     version: 8,
-    sources: {},
+    sources: {
+      esri: {
+        type: '',
+        scheme: '',
+        tilejson: '',
+        format: '',
+        maxzoom: 15,
+        tiles: [],
+        name: '',
+      },
+    },
     layers: [],
   };
-  const [mapState, setMapState] = React.useState({
+  const [mapState, setMapState] = React.useState<MapboxOptions>({
     mapStyle: EMPTY_STYLE,
```

Comment:
Argument of type mapStyle: is not assignable to parameter of type 'MapboxOptions | (() => MapboxOptions)'.
: The following warning is being displayed.

Reply:
this issue has been resolved